library/ui/src/main/java/com/google/android/exoplayer2/ui/PlayerView.java

```diff
@@ -564,6 +570,16 @@ public Bitmap getDefaultArtwork() {
    * @param defaultArtwork the default artwork to display.
    */
   public void setDefaultArtwork(Bitmap defaultArtwork) {
+    setDefaultArtwork(new BitmapDrawable(getResources(), defaultArtwork));
+  }
+
+  /**
+   * Sets the default artwork to display if {@code useArtwork} is {@code true} and no artwork is
+   * present in the media.
+   *
+   * @param defaultArtwork the default artwork to display
+   */
+  public void setDefaultArtwork(Drawable defaultArtwork) {
```

Comment:
Minor comment - Let's deprecate the method above. We should leave it there to avoid an unnecessary breaking change, but it becomes a little confusing because getDefaultArtwork will no longer pass back what the app has passed in.